docroot/themes/humsci/humsci_basic/src/scss/elements/_details.scss

```diff
@@ -0,0 +1,90 @@
+details {
+  margin: hb-calculate-rems(16px) 0;
+}
+
+summary {
+  padding: hb-calculate-rems(16px);
+  position: relative;
+  font-weight: hb-theme-font-weight('semibold');
+  cursor: pointer;
+
+  @include hb-colorful {
+    border: $hb-thin-border;
+    @include hb-pairing-color('border-color', 'secondary');
+    @include hb-pairing-color('background-color', 'secondary');
+    @include hb-global-color('color', 'white');
+  }
+
+  // This is to position everything correctly in browsers that do not support the
+  // -webkit-details-marker pseudo element
+  display: flex;
+  align-items: center;
+
+  // Removes the default arrow icon
+  &::-webkit-details-marker{
```

Comment:
awesome! can you include a comment about how this is for screen readers? thank you!!